src/main/scala/play/api/libs/mailer/MailerPlugin.scala

```diff
@@ -182,6 +185,7 @@ abstract class CommonsMailer(smtpHost: String, smtpPort: Int,
     email.setSubject(data.subject)
     email.setFrom(data.from)
     data.replyTo.foreach(setAddress(_) { (address, name) => email.addReplyTo(address, name)})
+    data.bounceAddress.map(email.setBounceAddress)
```

Comment:
Can you use `data.bounceAddress.foreach(email.setBounceAddress)` to stay consistent with the rest of the code ?